plugins/inventory/scaleway.py

```diff
@@ -13,6 +13,8 @@
     short_description: Scaleway inventory source
     description:
         - Get inventory hosts from Scaleway.
+    requirements:
+        - PyYAML
```

Comment:
This isn't a breaking change since Ansible/ansible-core already depends on PyYAML.